OMG.Classroom/DataAccessLayer/Seeder/Seed.cs

```diff
@@ -0,0 +1,308 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataAccessLayer.Seeder
+{
+    public class Seed
+    {
+        #region roles, students teachers
+        public static async Task SeedRolesAsync(ClassroomDbContext context)
+        {
+            if (await context.Roles.AnyAsync()) return;
+
+            var roles = new List<Entities.Role>
+            {
+                new Entities.Role
+                {
+                    //Id = Guid.NewGuid(),
```

Comment:
Set the ID to some random generated value that is hardcoded(!!!). Use online random guid generators for that